tests/functional/metrics/test_metric_helper_functions.py

```diff
@@ -3,30 +3,35 @@
 from dbt.tests.util import run_dbt, get_manifest
 from dbt.contracts.graph.metrics import ResolvedMetricReference
 
-from tests.functional.metrics.fixtures import models_people_sql, basic_metrics_yml
+from tests.functional.metrics.fixtures import (
+    models_people_sql,
+    basic_metrics_yml,
+    semantic_model_people_yml,
+    metricflow_time_spine_sql,
+)
 
 
 class TestMetricHelperFunctions:
     @pytest.fixture(scope="class")
     def models(self):
         return {
             "metrics.yml": basic_metrics_yml,
+            "semantic_people.yml": semantic_model_people_yml,
+            "metricflow_time_spine.sql": metricflow_time_spine_sql,
             "people.sql": models_people_sql,
         }
 
-    @pytest.mark.skip(
-        "TODO reactivate after we begin property hydrating metric `depends_on` and `refs`"
-    )
-    def test_expression_metric(
+    def test_derived_metric(
         self,
         project,
     ):
 
         # initial parse
-        run_dbt(["compile"])
+        run_dbt(["parse"])
 
         # make sure all the metrics are in the manifest
         manifest = get_manifest(project.project_root)
```

Comment:
small nit: I think we can remove this line in favor of grabbing the manifest from the result of `run_dbt(["parse"])`, e.g. `manifest = run_dbt(["parse"])`

Reply:
👍 Will do!